lib/warder/code_duplication_runner.rb

```diff
@@ -11,7 +11,13 @@ class CodeDuplicationRunner < Runner
     private
 
     def command_with_options
-      "#{COMMAND_NAME} -d -m #{FLAY_SCORE} #{@options.files}"
+      "#{COMMAND_NAME} -d -m #{flay_score} #{@options.files}"
+    end
+
+    def flay_score
+      env_flay_score = ENV['FLAY_SCORE']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.